wallet/how-to/send-transactions/send-batch-transactions.md

```diff
@@ -0,0 +1,204 @@
+---
+description: Send atomic batch transactions using `wallet_sendCalls`.
+---
+
+# Send batch transactions
+
+You can send and manage batch transactions in MetaMask, using the methods specified by
+[EIP-5792](https://eips.ethereum.org/EIPS/eip-5792):
+
+- `wallet_getCapabilities` - Query whether support for atomic batch transactions is available.
+- `wallet_sendCalls` - Submit multiple transactions to be processed atomically by MetaMask.
+- `wallet_getCallsStatus` - Track the status of your transaction batch.
+
+## About atomic batch transactions
```

Comment:
Content gives a good explanantion of the benefits of atomic batch transactions. But does not clearly define what an atomic batch transaction is. So perhaps add an intro regarding this?